pkg/collector/corechecks/snmp/report/report_device_metadata.go

```diff
@@ -73,11 +73,21 @@ func buildMetadataStore(metadataConfigs checkconfig.MetadataConfig, values *valu
 				}
 			} else {
 				for _, symbol := range symbols {
-					metricValues, err := getColumnValueFromSymbol(values, symbol)
+					indexValues := getColumnValueFromSymbolOrEmpty(symbol.IndexFromOidValue.OID, values)
```

Comment:
This part is a bit hard to grasp I feel, do you see some quick-wins we could get for readability for the future?
At the very least I feel like `getColumnValueFromSymbolOrNil` would be more explicit that the return value will be null.